### ISSUE TITLE
[Bug]: It won't run on Android

### ISSUE THREAD

**Issue by joaquingrech** [OneSignal/OneSignal-Unity-SDK#470]
### What happened?

Well well well, today i found that it doesn't run on iOS, Amazon and now neither in Android. Awesome!

So here is this bug, using latest sdk and Unity 2021.2.18f... for over a year now you can't place things into the /plugin/android/res or /asset folder. So I wonder what system do you use to test.

If you place anything on android on those folders, unity will fail to compile any project with a message like
OBSOLETE - Providing Android resources in Assets/Plugins/Android/assets was removed
or
OBSOLETE - Providing Android resources in Assets/Plugins/Android/res was removed

So what's the problem? Well, the problem is that if you mark the option when you import the Android package of importing the "res" folder, which includes the icons that you need to modify for your notification... your project will immediately fail to compile and run.

Same issue with Amazon and the api_key.txt file which is also required for it to run. But I created another bug with that one.

I'll give it a few days otherwise I'm removing OneSignal from my Unity project since it seems nobody is testing these things or taking them seriously there.

### Steps to reproduce?

```Markdown
1. Import android package.
2. Select to import the "res" icons so you can modify them.
3. Build and run on your device. Error.
```

### What did you expect to happen?

That you can build and test it.

### Unity version

2021.2.18f

### OneSignal Unity SDK version

3.0.0

### Platform

Android

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by adamschlesinger** [comment]
Hi @joaquingrech. We've pushed a fix for this in [#466](https://github.com/OneSignal/OneSignal-Unity-SDK/pull/466) which will be available in the 3.0.1 release.

**Comment by adamschlesinger** [comment]
Fixed by [#466](https://github.com/OneSignal/OneSignal-Unity-SDK/pull/466) in [Release 3.0.1](https://github.com/OneSignal/OneSignal-Unity-SDK/releases/tag/3.0.1).
